libraries/botbuilder-expression-parser/tests/expression.test.js

```diff
@@ -91,6 +92,27 @@ const dataSource = [
   ["trim(' hello ')", "hello"],
   ["trim(' hello')", "hello"],
   ["trim('hello')", "hello"],
+  ["endsWith('hello','o')", true],
+  ["endsWith('hello','a')", false],
+  ["endsWith(hello,'o')", true],
+  ["endsWith(hello,'a')", false],
+  ["startsWith('hello','h')", true],
+  ["startsWith('hello','a')", false],
+  ["countWord(hello)", 1],
+  ["countWord(concat(hello, ' ', world))", 2],
+  ["addOrdinal(11)", "11th"],
+  ["addOrdinal(11 + 1)", "12th"],
+  ["addOrdinal(11 + 2)", "13th"],
+  ["addOrdinal(11 + 10)", "21st"],
+  ["addOrdinal(11 + 11)", "22nd"],
+  ["addOrdinal(11 + 12)", "23rd"],
+  ["addOrdinal(11 + 13)", "24th"],
+  ["addOrdinal(-1)", "-1"],//original string value
+  ["count(guid())", 36],
+  ["guid().indexOf('-')", 8],
+  ["indexOf(guid(), '-')", 8],
+  ["guid().lastIndexOf('-')", 23],
+  ["lastIndexOf(guid(), '-')", 23],
```

Comment:
add `indexof` and `lastIndexOf` tests that return -1

Reply:
fixed.